### ISSUE TITLE
Can finally confirm iDevice not being recognized is a Bootra1n issue, not Checkra1n

### ISSUE THREAD

**Issue by carlgo18** [foxlet/bootra1n#94]
I've tested bootra1n on three devices:

-laptop with i5-5500U
-laptop with i7-7700
-PC with R5-2600x

On all instances, my iPad was not recognized by bootra1n. Reformatted my drive then switched from bootra1n to checkn1x (similar to bootra1n, albeit smaller in size). Checkn1x was able to recognize instantly, with no troubleshooting whatsoever and was able to jailbreak my device first try!

Hope this helps

**Comment by DEALWITHIT199** [comment]
Yes! I just tried checkn1x and it successfully worked! Bootra1n is still in beta ok thats why some devices cannot recognize.
